pkg/volume/gce_pd/attacher.go

```diff
@@ -96,6 +96,40 @@ func (attacher *gcePersistentDiskAttacher) Attach(spec *volume.Spec, nodeName ty
 	return path.Join(diskByIdPath, diskGooglePrefix+pdName), nil
 }
 
+func (attacher *gcePersistentDiskAttacher) VolumesAreAttached(specs []*volume.Spec, nodeName types.NodeName) (map[*volume.Spec]bool, error) {
```

Comment:
Might be time to create the Volumes interface on the cloudprovider :-)  The objection last time was that there could be different types; maybe if the volumes interface took a string argument for the "volume type" that would overcome that objection?  cc @thockin